scripts/lib/spreadsheet-standardizer.ts

```diff
@@ -33,10 +36,12 @@ export class SpreadsheetStandardizer {
     valueMap: { [index: string]: AliasMap } = VALUE_MAPPINGS,
     strict: boolean = true,
   ) {
+    this.fieldMap = fieldMap;
```

Comment:
I think it might be worth adding a comment about the uses of the field/val maps, and their reversed maps. AFAICT the original maps are now saved due to wanting to convert back to aliases? but since the original names were swapped out, it took a bit.

Reply:
Fair, this was probably a pretty awkward diff given the original names weren't great. Added some comments above the constructor.

Reply:
Does this need to refresh? I don't see the comments.

Reply:
Oops, I was on the former commit. LGTM!